source/style-guide/style/placeholder-variable-text.txt

```diff
@@ -26,13 +28,79 @@ name of a directory.
    the path. For information about formatting environment variables,
    see :ref:`text-formatting`.
 
+Placeholder Text Guidelines
+---------------------------
+
 When creating placeholder text, use the following guidelines.
 
+Apply Language-Specific Formatting
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 20 20 60
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Language
+     - Format
+     - Additional Guidelines
+
+   * - Shell, PowerShell, Command Prompt
+     - ``<variable-name>``
+     - Follow `docopt <http://docopt.org/>`__
+       guidelines
+
+   * - API
```

Comment:
Nitpick: since you separated "API Resources", this could be "API parameters" to make the difference more clear.